SPECS/xapi-nbd.spec

```diff
@@ -12,6 +12,9 @@ Source2: SOURCES/xapi-nbd/xapi-nbd.path
 
 Provides: gitsha(https://code.citrite.net/rest/archive/latest/projects/XSU/repos/xapi-nbd/archive?at=v1.11.0&format=tar.gz&prefix=xapi-nbd-1.11.0#/xapi-nbd-1.11.0.tar.gz) = 40168edbe1857e993f417af5af554764e751a9e0
 
+# XCP-ng patches
+Patch1000: xapi-nbd-1.11.0-adapt-vbd.create-call.XCP-ng.patch
```

Comment:
Please add a comment telling in what version of xapi-ndb this patch is not necessary anymore, so that we remember we can drop it in 8.3

Reply:
E.g.
```
# Patch not necessary anymore in xapi-nbd 1.12.0 and above